examples/demo-rollup/README.md

```diff
@@ -192,37 +192,33 @@ pub enum CallMessage<C: sov_modules_api::Context> {
     },
 }
 ```
-In the above snippet, we can see that `CallMessage`s in `Bank` support five different types of calls. The `sov-cli` has the ability to parse a JSON file that aligns with any of these calls and subsequently serialize them. The structure of the JSON file, which represents the call, closely mirrors that of the Enum member. Consider the `CreateToken` message as an example:
+In the above snippet, we can see that `CallMessage`s in `Bank` support five different types of calls. The `sov-cli` has the ability to parse a JSON file that aligns with any of these calls and subsequently serialize them. The structure of the JSON file, which represents the call, closely mirrors that of the Enum member. Consider the `Transfer` message as an example:
```

Comment:
I just saw that @neysofu also changed this file in this PR: https://github.com/Sovereign-Labs/sovereign-sdk/pull/477

Reply:
Right – I think getting the fix out asap is more important though, @dubbelosix feel free to merge first and I'll fix the conflicts later.